README.md

```diff
@@ -363,6 +363,7 @@ Additional lists you might find useful:
 - [Cakebox](https://github.com/alt3/cakebox) - A Vagrant development environment powered by the CakePHP 3.x Console.
 - [CakePHP.gitignore](https://github.com/github/gitignore/blob/master/CakePHP.gitignore) - The .gitignore file proposals.
 - [PhpStorm plugin](https://github.com/skie/PhpStorm) -  CakePHP auto-completion support for console commands in PhpStorm IDE.
+- [NetBeans plugin](https://github.com/junichi11/cakephp3-netbeans#netbeans-cakephp3-plugin-dev) -  This plugin provides support for CakePHP3 in NetBeans 8.1+.
```

Comment:
But in general it is also wiser to keep versions out of the URL an Name in general. To be future proof.